src/views/shared/SecurityBeacon.js

```diff
@@ -120,7 +140,8 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
         <div class="circle right"></div>\
       </div>\
     </div>\
-    <div class="bg-helper auth-beacon auth-beacon-security" data-se="security-beacon">\
+    <div aria-live="polite" role="alert" class="bg-helper auth-beacon auth-beacon-security" data-se="security-beacon">\
```

Comment:
I can change this, but when I use `aria-live="polite"`, I never get the text read by Voiceover.

That said, without having a screen reader expert to consult, I am ok with doing exactly what the customer asked for, under the assumption that they know something about their tools that I do not know.